core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/SchemaExport.java

```diff
@@ -15,8 +15,22 @@
  */
 package org.mobilitydata.gtfsvalidator.annotation;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
 /**
  * Annotation to be used on notice constructor. This specifies the constructor to be used to
- * identify notice field names and data types when exporting the notice schema.
+ * identify notice field names and data types when exporting the notice schema. This annotation
+ * defines new variables to be used when export the notice schema as a json file.
  */
-public @interface SchemaExport {}
+@Retention(RetentionPolicy.RUNTIME)
+public @interface SchemaExport {
+  String JSON_KEY_NAME = "name";
+  String JSON_KEY_TYPE = "type";
+  String BLOB_DATA_TYPE = "BLOB";
+  String VARCHAR_DATA_TYPE = "VARCHAR";
+  String BIGINT_DATA_TYPE = "BIGINT";
+  String INTEGER_DATA_TYPE = "INTEGER";
+  String CLASS_SIMPLE_NAME_SEVERITY_LEVEL = "SeverityLevel";
```

Comment:
What are those constants doing in this class? Can they be moved to a place where they are actually used?

Reply:
I was addressing [this review comment](https://github.com/MobilityData/gtfs-validator/pull/925#discussion_r674420430). Will be moved to `NoticeSchemaGenerator.java` after creation.